runtime/directives/sky-app-link-external.directive.ts

```diff
@@ -0,0 +1,36 @@
+import { Directive, Input, HostListener, OnInit } from '@angular/core';
+import { SkyAppWindowRef } from '../window-ref';
+import { SkyAppConfig } from '@blackbaud/skyux-builder/runtime';
+import { HttpParams } from '@angular/common/http';
+
+@Directive({
+  selector: '[skyAppLinkExternal]'
+})
+export class SkyAppLinkExternalDirective implements OnInit {
+  @Input() private skyAppLinkExternal: string;
+  private externalSPAUrl: string;
```

Comment:
To remain consistent with terminology used across the rest of the app, this should be named `externalAppUrl` or simply `externalUrl`.